tools/generate-docs

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+# Ensure this script must be executed on the project's root.
+SCRIPT_PATH="${BASH_SOURCE[0]:-$0}";
+cd "$( dirname "$( dirname -- "$( realpath "$SCRIPT_PATH" )" )" )"
```

Comment:
why not just `cd "$SCRIPT_PATH/.."`?

Reply:
I'll re-quote-link myself on this one:

https://github.com/os-autoinst/openQA/pull/5389/files#r1422496164

Reply:
I am not convinced why tools/generate-docs should use a different implementation in relation to other tools just based on personal preference to make something "more explicit". Either that's the way to go for all cases or we stick to the existing way.

Reply:
> I am not convinced why tools/generate-docs should use a different implementation in relation to other tools just based on personal preference to make something "more explicit". Either that's the way to go for all cases or we stick to the existing way.

What do you mean by different implementation in relation to other tools? Can you be more explicit (no joke intended)

> Either that's the way to go for all cases or we stick to the existing way.

Lets be explicit :) overall

Reply:
I mean other uses of `BASH_SOURCE` in the scope of https://github.com/os-autoinst

Reply:
That will mean I'll have to go over *all of os-autoinst* to enforce explicitness on bash scripts.

That's too much effort, sorry. Applied @baierjan suggestion.

Reply:
> I mean other uses of `BASH_SOURCE` in the scope of https://github.com/os-autoinst

@okurz please add a style check of such things and/or add it to a contributing guide, it can't be that conversations and contributions are stalled for days, because of arbitrary standards or preferences (i.e the Ternay operator discussion et al)